client/settings/payment-settings/account-keys-modal.js

```diff
@@ -131,11 +132,42 @@ const TestWebhookSecret = () => {
 	);
 };
 
+const WebhookEndpointText = styled.strong`
+	padding: 0 2px;
+	background-color: #f6f7f7; // $studio-gray-0
+`;
+
+const WebhookInformation = () => {
+	const { data } = useAccount();
+	return (
+		<p>
+			{ interpolateComponents( {
+				mixedString: __(
+					"Add the following webhook endpoint {{webhookUrl/}} to your {{settingsLink}}Stripe account settings{{/settingsLink}} (if there isn't one already). This will enable you to receive notifications on the charge statuses.",
+					'woocommerce-gateway-stripe'
+				),
+				components: {
+					webhookUrl: (
+						<WebhookEndpointText>
+							{ data.webhook_url }
+						</WebhookEndpointText>
+					),
+					settingsLink: (
+						// eslint-disable-next-line jsx-a11y/anchor-has-content
+						<a href="https://dashboard.stripe.com/account/webhooks" />
+					),
+				},
+			} ) }
+		</p>
+	);
+};
+
```

Comment:
While adding this new text, we're reaching three instances of the same piece of code, I think now is a good moment to consider extracting that to a new component and make it reusable.

Reply:
Fixed in 111a8a521ce355312f6ea89610f9c6a4ca4d7129 🙂